src/uint/traits.rs

```diff
@@ -220,3 +220,31 @@ impl<const L: usize> BoxedEncoding for Uint<L> {
         Ok(Self::from_be_slice(bytes))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crypto_bigint::U256;
+    use sha3::Shake256;
+
+    use super::FromXofReader;
+    use crate::tools::hashing::{Chain, Hasher};
+
+    #[test]
```

Comment:
```suggestion
    // Test to ensure that random seeds work the same on 32 and 64 bit platforms
    #[test]
```

…or something to that effect.